crates/polars-io/src/parquet/write.rs

```diff
@@ -98,7 +98,7 @@ pub struct ParquetWriter<W> {
     compression: CompressionOptions,
     /// Compute and write column statistics.
     statistics: bool,
-    /// if `None` will be 512^2 rows
+    /// if `None` will equal the number of rows in the data frame
```

Comment:
I don't want this change. We should keep the default of 512^2 rows. We don't mean to copy pyarrow.

Reply:
> We don't mean to copy pyarrow.

Fair enough :smile: I think it would be nice to have a way of specifying "one row group" though, at least in the Python interface.

> We should keep the default of 512^2 rows

Out of curiosity, what's the benefit of this paired with the default `statistics=False`? Wouldn't `polars` be required to read all row groups even in the presence of filters?

Reply:
Since we usually work with lazy frames (that cannot be serialized with `sink_parquet`), we would need to assign a variable, i.e.:

```python
df = lf.collect()
df.write_parquet(row_group_size=len(df))
```

rather than

```python
lf.collect().write_parquet(row_group_size=None)
```